lib/exfsm.ex

```diff
@@ -107,6 +109,23 @@ defmodule ExFSM do
       def unquote(signature), do: unquote(body_block[:do])
     end 
   end
+
+  @doc """
+    Define a function of type `multitransition` describing a transition and its
+    different states or not. The function name is the transition name, the mode (:excluded or :included) is the
+    first argument with a list of the states linked to this mode. A state object can be modified and is the second argument.
+        defmultitrans refund({:included, [:payed, :shipped, :partially_shipped], _}, order) do
+          {:next_state, :refunded, order}
+        end
+  """
+  defmacro defmultitrans({event,_meta,[{_, _, [mode, states, _params]}| _rest]}=signature,body_block) do
+    quote do
+      @multitrans Map.put(@multitrans,{unquote(event), unquote(states), unquote(mode)},__MODULE__)
+      doc = Module.get_attribute(__MODULE__, :doc)
+      @docs Map.put(@docs,{:event_doc,unquote(event)},doc)
+      def unquote(signature), do: unquote(body_block[:do])
+    end
+  end
```

Comment:
Is there a reason you don't handle the `to` attribute ? like with `deftrans`

Reply:
I just took the `defbypass` as model because at the beginning it was more similar to the `defbypass`.
There isn't any particular reason.